views/MapsContainer.tsx

```diff
@@ -16,17 +20,25 @@ class MapsContainer extends React.PureComponent<MapsContainerProps> {
             <Maps
                 gpsLocation={this.props.gps.data.at(-1) || {}}
                 gpsPath={this.props.gps.data.map(
-                    (gpsCoordinate: GPSCoordinate) => convertToLatLng(gpsCoordinate)
+                    (gpsPoint: GPS) => convertToLatLng(gpsPoint)
+                )}
+                globalPath={this.props.globalPath.data.waypoints.map(
+                    (waypoint: GlobalPath) => convertToLatLng(waypoint)
                 )}
                 aisShips={this.props.aisShips.data.ships}
+                localPath={this.props.localPath.data.waypoints.map(
+                    (waypoint: LocalPath) => convertToLatLng(waypoint)
```

Comment:
I think the import for type `LocalPath` is missing here

Reply:
done